tests/specs/graph/fast_check/expando_props/funcs_ref_private_name.txt

```diff
@@ -8,7 +8,7 @@
 export function test() {
 }
 
-test.skip = (nonExported: NonExported) => {};
```

Comment:
Why do we add a return statement here?

Reply:
Do you want me to change it so we infer an empty function body to `void`?

Reply:
I like how the GH pull request extension caused this to comment on the wrong line (should have been the one below).

It just seems unnecessary (though not a big deal) to have an empty return in the output.